.travis.yml

```diff
@@ -1,15 +1,5 @@
 language: python
 
-services:
```

Comment:
We have overhauled the travis file in mdstore as a result of adding authentication to mongo shells. Please check what I have done [here](https://github.com/arkilic/metadatastore/blob/authenticate_mongoserv/.travis.yml)